azure-pipelines/build.yml

```diff
@@ -18,38 +18,13 @@ steps:
   inputs:
     useGlobalJson: true
 
-- task: DotNetCoreCLI@1
-  displayName: 'dotnet restore src/Microsoft.SqlTools.Credentials'
-  inputs:
-    command: restore
-    projects: '$(Build.SourcesDirectory)/src/Microsoft.SqlTools.Credentials'
-    arguments: '--configfile $(Build.SourcesDirectory)/nuget.config'
-
-- task: DotNetCoreCLI@1
-  displayName: 'dotnet restore src/Microsoft.SqlTools.Hosting'
-  inputs:
-    command: restore
-    projects: '$(Build.SourcesDirectory)/src/Microsoft.SqlTools.Hosting'
-    arguments: '--configfile $(Build.SourcesDirectory)/nuget.config'
-
-- task: DotNetCoreCLI@1
-  displayName: 'dotnet restore src/Microsoft.SqlTools.ServiceLayer'
-  inputs:
-    command: restore
-    projects: '$(Build.SourcesDirectory)/src/Microsoft.SqlTools.ServiceLayer'
-    arguments: '--configfile $(Build.SourcesDirectory)/nuget.config'
-
-- task: DotNetCoreCLI@1
-  displayName: 'dotnet restore src/Microsoft.Kusto.ServiceLayer'
-  inputs:
-    command: restore
-    projects: '$(Build.SourcesDirectory)/src/Microsoft.Kusto.ServiceLayer'
-    arguments: '--configfile $(Build.SourcesDirectory)/nuget.config'
-
 - task: DotNetCoreCLI@2
-  displayName: 'dotnet build src/Microsoft.SqlTools.ServiceLayer'
+  displayName: 'dotnet restore sqltoolsservice.sln'
```

Comment:
is .sln a newly supported target for restore and build operation? wondering why we we doing not doing that before

Reply:
No it isn't. No clue why we weren't doing it before - maybe whoever made it originally only cared about building specific projects. But I'd rather we just simplify this since we should be building everything anyways.